bin/main.rb

```diff
@@ -1,2 +1,29 @@
-#!/usr/bin/env ruby
-puts 'Hello world!'
+require_relative '../lib/Player'
+require_relative '../lib/Game'
+
+puts 'Please enter player1 name: '
+player_1_name = gets.chomp
+player1 = Player.new(player_1_name, 'x')
+
+puts 'Please enter player2 name: '
+player_2_name = gets.chomp
+player2 = Player.new(player_2_name, 'o')
+
+game = Game.new(player1, player2)
+
+puts "Welcome to the Tic - Tac - Toe game #{player1.name} and #{player2.name}"
+
+loop do
+  game.move(player1)
+  unless game.any_empty?
+    puts "It's draw!"
+    break
+  end
+  puts ''
+  puts '- - - - - - - -'
+  game.move(player2)
+  unless game.any_empty?
+    puts "It's draw!"
+    break
```

Comment:
- [ ] Is this checking for the win? You need to at least put something that you will do it. Remember that it does not need to be fully working yet.